x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/operator/MultivalueDedupeBytesRef.java

```diff
@@ -317,6 +340,25 @@ private void writeSortedWork(BytesRefBlock.Builder builder) {
         builder.endPositionEntry();
     }
 
+    /**
+     * Writes a {@link #work} to a {@link BytesRefBlock.Builder}.
+     */
+    private void writeWork(BytesRefBlock.Builder builder, boolean ascending) {
```

Comment:
Maybe rename `writeStoredWork` to `deduplicateSortedWork` or something. Because this method is writing the sorted work.